frontend/tslint.json

```diff
@@ -6,9 +6,10 @@
   "rules": {
     "array-type": false,
     "arrow-parens": false,
-    "deprecation": {
-      "severity": "warn"
-    },
+    "deprecation": false, // TODO reenable when issue #29905 of angular is fixed
```

Comment:
I'm sad because I thought I fixed it, but I missed one deprecated in the API 😢 I have a pending PR that will remove the warning, hopefully it will be merged in 8.0.x

Reply:
Good to know, thanks.